buildpack/telemetry/telegraf.py

```diff
@@ -59,7 +59,18 @@ def _get_appmetrics_target():
 
 
 def include_db_metrics():
-    return strtobool(os.getenv("APPMETRICS_INCLUDE_DB", "true"))
+    if util.is_free_app():
+        # For free apps we are not interested in database metrics
+        return False
+
+    value = "false"
+    if _get_appmetrics_target() is not None or datadog.is_enabled():
+        # For customers who have Datadog or APPMETRICS_TARGET enabled,
+        # we always include the database metrics. They can opt out
+        # using the APPMETRICS_INCLUDE_DB flag
+        value = "true"
```

Comment:
I think its better if the APPMETRICS_INCLUDE_DB check is inside this if block, so at the end you can simply return strtobool(value)

Reply:
I've included the env var check inside the block now.